.travis.yml

```diff
@@ -51,7 +51,7 @@ jobs:
 
   # Run build and tests on ARM64 on Pull Requests.
```

Comment:
You might want to add a short description of why the split on tests is taking place

Reply:
:+1: Done with https://github.com/etclabscore/core-geth/pull/328/commits/f466a2e1408967d048ccbb798d06b8e1dd88e6ed